packages/build/tests/plugins/tests.js

```diff
@@ -1279,83 +1279,33 @@ test('Does not pin netlify.toml-only plugin versions if there are no site ID', a
   await runWithPluginRunsMock(t, 'pin_config_success', { flags: { siteId: '' } })
 })
 
-// For some tests of the `node-path` flag we have to disable the buildbot_build_plugins_system_node_version feature flag
-// as with it we always rely on the system node version (and is currently set to true by default)
-
-test('Validate --node-path version is supported by our codebase', async (t) => {
-  const nodePath = getNodePath('8.2.0')
-  await runFixture(t, 'node_version_simple', {
-    flags: { nodePath, featureFlags: { buildbot_build_plugins_system_node_version: false } },
-  })
-})
-
-test('No --node-path version validation is performed if the plugins_system_node_version feature flag is set', async (t) => {
-  const nodePath = getNodePath('8.2.0')
-  await runFixture(t, 'node_version_simple', {
-    flags: { nodePath, featureFlags: { buildbot_build_plugins_system_node_version: true } },
-  })
-})
-
```

Comment:
This test validated the warning message wasn't generated if the flag was enabled, hence no longer required.